CHANGELOG.md

```diff
@@ -3,7 +3,14 @@
 ## Upcoming
 
 - `apollo`
-  - <First `apollo` related entry goes here>
+  - https://github.com/apollographql/apollo-tooling/pull/1849
```

Comment:
Formatting of this entry isn’t consistent with what we have but I’m not concerned too much there, and that’s non blocking. I can amend before releasing since I’ll have to do that anyway